Razorpay.php

```diff
@@ -1,10 +1,13 @@
 <?php
 
-// Include Requests
-require_once __DIR__.'/libs/Requests-1.6.1/library/Requests.php';
+// Include Requests only if not already defined
+if (class_exists('Requests') === false)
+{
+    require_once __DIR__.'/libs/Requests-1.6.1/library/Requests.php';
 
-// Register requests autoloader
-Requests::register_autoloader();
+    // Register requests autoloader
+    Requests::register_autoloader();
+}
 
```

Comment:
We should throw an exception in the else clause giving clear indication of why it's failed otherwise, this may fail later in the code and it will be difficult to debug.